api/lib/schema.js

```diff
@@ -133,7 +133,7 @@ const typeDefs = gql`
     id: String
     title: String
     chain_id: String
-    rpc_url: String
+    public_rpc_url: String
```

Comment:
This is wrong right. Why do you delete `rpc_url` here. Now you get the following error:

![image](https://user-images.githubusercontent.com/40721795/85599709-0fadb200-b64d-11ea-9da3-0c66db8d6966.png)

Reply:
let's revert I changed it actually to return the public url